LR_Tourist/BLL/BLL.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="9.0.0" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Ninject" Version="3.3.4" />
+    <PackageReference Include="NLog" Version="4.7.0" />
+    <PackageReference Include="NLog.Config" Version="4.7.0" />
+    <PackageReference Include="Scrutor" Version="3.2.0" />
```

Comment:
This package should be not used here.
It should be used only in case of using Microsoft DI and only in `Tourist` project.

Reply:
fixed

Reply:
It still not resolved.
`Scrutor` package should be in `Tourist` project.
Also, `AutoMapper.Extensions.Microsoft.DependencyInjection` also should be in `Tourist` project.